kotlin/kotlin-quiz.md

```diff
@@ -34,7 +34,7 @@ class Airplane(private val name: String) : Aircraft() {
 - [ ] Only abstract classes are inheritable by subclasses
 - [ ] Only abstract classes can inherit from multiple superclasses
 - [ ] Only abstract classes can have abstract methods
-- [ ] Only abstract classes can store state
+- [x] Only abstract classes can store state
```

Comment:
@adminazhar could you please add your reference link with answer here, would be very useful for others! Thanks, have a good day!

Reply:
Okay